css/aboutus.css

```diff
@@ -24,7 +24,7 @@
 .about p{
     width: 90%;
     margin: auto;
-    padding: 5px 0px 5px 0px;
+    padding: 5px auto;
```

Comment:
I apologize, my previous comment said `5px auto` it should have been `5px 0px` thats on me

Reply:
file adjusted